### DISCUSSION TITLE
Trace And Optimizeを使用したときにパーティクル系ギミックのスケールがおかしくなる

### DISCUSSION THREAD

**Discussion by yume327** [anatawa12/AvatarOptimizer#1196] [Q&A] [answered]
はじめまして。
だまこやさんの【VRC想定】カメラエフェクト / Camera Effects（ https://kodamaaltanative.booth.pm/items/5855210 ）を
Trace And Optimizeコンポーネントを使用しているアバターに導入した際、Play/VRChatへのアップロードをするとパーティクルのスケールが意図しないサイズになってしまいました。
特に顕著だったのがレンズフレア風パーティクルです
![スクリーンショット 2024-09-22 235023](https://github.com/user-attachments/assets/3b46cbe1-b26d-4189-a60f-9955ad792e37)
![スクリーンショット 2024-09-22 235309](https://github.com/user-attachments/assets/6dd1236b-e403-4a66-9bdb-c896276ba2fb)
(うおっ眩しっ)

Trace And Optimizeの「使われていないObjectを自動的に削除する」のチェックをはずすと本来のスケールで再生できるのですが、考えられる原因はありますでしょうか？(なんとなく空オブジェクトのスケールが関係していそうな気はしますが……)
以下VCCの画面と、シーン上で関係していそうなオブジェクトのスクリーンショットを添付します。
他に必要な情報があれば提示しますので教えてください。
![スクリーンショット 2024-09-23 000100](https://github.com/user-attachments/assets/5902e4f2-6b85-4cb0-9182-2843b1b385e4)
![スクリーンショット 2024-09-22 235438](https://github.com/user-attachments/assets/00672957-d9f4-4f4e-a661-6a98a79bd4e9)
![スクリーンショット 2024-09-22 235007](https://github.com/user-attachments/assets/f1cb35a1-1f94-4228-8614-c68e0c70c339)
(Lens Flare_xxの各オブジェクトスケールはすべて同じ)

ご確認のほど、よろしくお願いします。

**Comment by anatawa12** [comment]
少し調べてみた感じScaling Mode Localの対応に問題がありそうですね。確認します。

**Reply by anatawa12** [reply]
#1197 にしました。

**Comment by anatawa12** [comment, marked as answer]
1.7.13-beta.1 で修正できたと思います。確認お願いします。

**Reply by yume327** [reply]
該当バージョンでの修正確認しました。問題なさそうです。ご対応ありがとうございます！
